src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -210,6 +199,10 @@ public string HubName
         // Used for mocking the lifecycle notification helper.
         internal HttpMessageHandler NotificationHandler { get; set; }
 
+        // This is purely a way for tests to overwrite the webhook url, since there is no easyway
+        // to mock the value from ExtensionConfigContext.
```

Comment:
minor nit:
```suggestion
        // This is just a way for tests to overwrite the webhook url, since there is no easy way
        // to mock the value from ExtensionConfigContext.
```